src/Text/Pretty/Simple.hs

```diff
@@ -116,11 +122,16 @@ import Control.Applicative
 
 import Control.Monad.IO.Class (MonadIO, liftIO)
 import Data.Text.Lazy (Text)
-import Prettyprinter.Render.Terminal (renderLazy, renderIO)
+import Prettyprinter (SimpleDocStream)
+import Prettyprinter.Render.Terminal
+      (Color (..), Intensity(Vivid,Dull), AnsiStyle,
+       renderLazy, renderIO)
```

Comment:
I haven't quite been able to work out the formatting rules for imports. Do you use a particular formatter?

Reply:
`hindent` used to format imports like this (but now it has been changed to format import lists differently).

However, if you want to change this (or even run a formatter over all the code), please feel free.